app/assets/stylesheets/newflow.scss

```diff
@@ -4,6 +4,8 @@ $font-path: '/accounts/fonts';
 @import "font-awesome";
 @import "newflow_colors";
 @import "intlTelInput";
+@import "radio_button";
+@import "radio_buttons";
```

Comment:
`radio_button` and `radio_buttons`? Looks like a typo, but is it? If not, please add a comment why.